pypolygames/tests/test_zoo.py

```diff
@@ -6,11 +6,42 @@
 from unittest import SkipTest
 import pytest
 import torch
+import torch.nn.functional as F
 from .. import model_zoo
 from ..model_zoo.utils import get_game_info
 from .. import params
 from .. import utils
 
+# NOTE: Copied this loss from a model in the first commit
```

Comment:
not sure what "first commit" refers to. Maybe comment should paste a GitHub link to the commit & line of code? e.g., link to https://github.com/facebookarchive/Polygames/blob/eb5390e57cc38e5287bf6dcfb420308a5995d194/pypolygames/model_zoo/amazons_model.py#L135 or whatever it is you copied this from